sim/core/buffs.go

```diff
@@ -65,8 +65,10 @@ func applyBuffEffects(agent Agent, raidBuffs *proto.RaidBuffs, partyBuffs *proto
 		ThornsAura(character, 0)
 	}
 
-	if raidBuffs.MoonkinAura > 0 || raidBuffs.ElementalOath {
+	if raidBuffs.MoonkinAura > 0 || raidBuffs.ElementalOath == proto.TristateEffect_TristateEffectImproved {
 		character.AddStat(stats.SpellCrit, 5*CritRatingPerCritChance)
+	} else if raidBuffs.ElementalOath == proto.TristateEffect_TristateEffectRegular {
+		character.AddStat(stats.SpellCrit, 3*CritRatingPerCritChance)
 	}
```

Comment:
Moonkin aura now provides 5% haste and no crit. You could probably remove it from here. https://github.com/wowsims/cata/pull/45 adds it to the group of haste effects